examples/particle.zig

```diff
@@ -1,14 +1,27 @@
 usingnamespace @import("QGuiApplication");
 usingnamespace @import("QQmlApplicationEngine");
+usingnamespace @import("QObject");
+usingnamespace @import("QVariant");
+
+const string = []u8;
 
 pub fn main() anyerror!void {
     QGuiApplication.init();
     defer QGuiApplication.quit();
 
     var engine = QQmlApplicationEngine.create();
     defer engine.delete();
-
     engine.loadData(@embedFile("particle.qml"));
 
+    var context = engine.rootContext();
+
     QGuiApplication.exec();
 }
+
```

Comment:
Still a major TODO is to pass an instance of Control to the QML side
Needs QObject() to be completed yet - and thats pretty tricky

working on it